tests/dummy/app/styles/app.css

```diff
@@ -16,3 +15,6 @@ body {
   list-style: none;
 }
 
+.an-excluded-class {
+    padding: 5px;
```

Comment:
Nitpick: change this to two space indentation. Also, please replace the `padding` with `background-color: #eee;` – just so the click area is more visible.